components/bl-markdown-component/src/index.js

```diff
@@ -0,0 +1,62 @@
+import MarkdownIt from './markdown-it.min.js';
```

Comment:
I'm not sure, but our eslint config has import order and relative imports should be under the libraries

Reply:
yes, I fixed it